core/appeal/service.go

```diff
@@ -1241,13 +1239,14 @@ func (s *Service) getAppealsMap(ctx context.Context, filters *domain.ListAppeals
 
 	appealsMap := map[string]map[string]map[string]*domain.Appeal{}
 	for _, a := range appeals {
-		if appealsMap[a.AccountID] == nil {
-			appealsMap[a.AccountID] = map[string]map[string]*domain.Appeal{}
+		accountID := strings.ToLower(a.AccountID)
+		if appealsMap[accountID] == nil {
+			appealsMap[accountID] = map[string]map[string]*domain.Appeal{}
```

Comment:
we need to make sure that `appealsMap` values are assigned using lowercase email as well